### DISCUSSION TITLE
Add the Fischer Tropsch Reactor to the game

### DISCUSSION THREAD

**Discussion by Technomirfuture132** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1139] [General]
I really love this mod, but it lacks realism, at the moment there are three branches of oil production in the game, there are biofuels, however, in modern petrochemistry there is the possibility of producing synthetic fuels from synthesis gas by Fischer-Tropsch synthesis. Could you add something like this to the game? The concept is that this reactor will produce cleaner fuel, but the components for it will require materials available closer to the nuclear era, similar to the catalytic reforming units already added to the game.

**Comment by Technomirfuture132** [comment]
In addition to the above, one method of producing syngas using an industrial mixer has already been added to the Hbm mod, but this method is not real or exceptional, I propose to introduce a gasification system - any substances containing carbon and hydrogen can serve as raw materials for gasification - biomass, natural gas, coal (wood or stone, brown) or petroleum waste. Steam and oxygen will be supplied to the plant, as well as solid raw materials, at the output we will get solid coal residue (coke) or ash, as well as synthesis gas. This gasification plant will have to produce a sufficient amount of synthesis gas for the aforementioned Fischer-Tropsch synthesis plant.

**Comment by JameH2** [comment]
Aside from realism, what would this bring into gameplay?

we have loads of petrochemical processing machines, most take automation to account (chlorine + ref gas = high-perf) so what can this offer?

**Reply by Technomirfuture132** [reply]
I think that alternative technologies for the production of liquid hydrocarbons should be added to the gameplay. The mod, as you know, adds a pollution system, new types of fuel obtained by Fischer-Tropsch synthesis will practically not produce any soot emissions, let it be an alternative oil branch. The petroleum products produced at the plant will be more energy-valuable, produce more heat, at about the same level as the reformed fuel, but will not produce any soot emissions. It is possible that several operating modes can be made for the reactor (at atmospheric pressure), medium and high pressure will result in different degrees of caloric content of fuel. All of them, unlike oil ones, will have a more transparent texture, since this is in reality, it could further diversify the ways for the production of polymers, synthetic paraffin and new explosives.
The raw material for production is synthesis gas, which could be obtained at a gasification plant from heterogeneous raw materials.
